Formula/kibana.rb

```diff
@@ -49,6 +49,7 @@ def install
     prefix.install Dir["build/kibana-#{version}-#{platform.sub("x64", "x86_64")}/{bin,config,node_modules,optimize,package.json,src,webpackShims}"]
 
     inreplace "#{bin}/kibana", %r{/node/bin/node}, "/libexec/node/bin/node"
+    inreplace "#{bin}/kibana-plugin", %r{/node/bin/node}, "/libexec/node/bin/node"
```

Comment:
Thank you for the suggestion @MikeMcQuaid. For now I would prefer to keep it explicit like this. I do know that right now there is nothing in the works that would lead to there being a third script here. If this changes in the future, I think your suggestion would be a better approach.